src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -30,11 +31,55 @@ function InboxReason({inbox, fontSize = 'sm'}: Props) {
       ? `More than ${Math.round(count / EVENT_ROUND_LIMIT)}k`
       : `${count}`;
 
+  function getTooltipDescription() {
+    const {until, count, window, user_count, user_window} = reason_details;
+    if (until) {
+      // Was ignored until `until` has passed.
+      //`until` format: "2021-01-20T03:59:03+00:00"
+      return tct('Was ignored until [window]', {
+        window: <DateTime date={until} dateOnly />,
```

Comment:
@priscilawebdev I think this is fine for the tooltip, we only need to show the day here and we can treat the date as timezone'd like we do in other places in the issue stream.

Reply:
yes. I think Datetime is fine here too :)